api/chalicelib/core/tenants.py

```diff
@@ -52,7 +53,15 @@ def edit_tenant(tenant_id, changes):
         return helper.dict_to_camel_case(cur.fetchone())
 
 
-def tenants_exists(use_pool=True):
+def tenants_exists_sync(use_pool=True):
     with pg_client.PostgresClient(use_pool=use_pool) as cur:
-        cur.execute(f"SELECT EXISTS(SELECT 1 FROM public.tenants)")
+        cur.execute("SELECT EXISTS(SELECT 1 FROM public.tenants)")
         return cur.fetchone()["exists"]
+
+
+async def tenants_exists(use_pool=True):
+    async with orpy.get().database.connection() as cnx:
+        async with cnx.transaction() as txn:
+            row = await cnx.execute("SELECT EXISTS(SELECT 1 FROM public.tenants)")
+            row = await row.fetchone()
```

Comment:
```suggestion
        async with cnx.transaction():
            await cnx.execute("SELECT EXISTS(SELECT 1 FROM public.tenants)")
            row = await cnx.fetchone()
```

Reply:
I committed, then reset hard that commit because it does not work. See `NOTES.md`.